alpha/scripts/utils/restoreMetadataObjects.php

```diff
@@ -0,0 +1,125 @@
+<?php
+ini_set('memory_limit','1024M');
+if ($argc < 3)
+{
+	echo PHP_EOL . ' ---- Restore Deleted Metadata Profile && Objects ---- ' . PHP_EOL;
+	echo ' Execute: php ' . $argv[0] . ' partnerId, metadata profile ID, [ /path/to/metadata object IDs list || metadataObjectId_1,metadataObjectId_2,.. || metadataObject ] [realrun / dryrun]' . PHP_EOL;
+	die(' Error: missing metadataObjectIds file, csv or single entry ' . PHP_EOL . PHP_EOL);
+}
+
+$partnerId = $argv[1];
+$metadataProfileId = $argv[2];
+$metadataObjectIds = explode(',', $argv[3]);
+$totalMetadataObjectIds = count($metadataObjectIds);
+
+
+require("/opt/kaltura/app/alpha/scripts/bootstrap.php");
+
+$dryRun = true;
+if (isset($argv[3]) && $argv[3] == 'realrun')
```

Comment:
Argument realrun/dryrun is now at argv[4]

Reply:
changed.